src/MCMCChain.jl

```diff
@@ -30,13 +35,11 @@ Parameters:
 - `names`: List of variable names (strings)
 - `chains`: List of chain ids
 """
-struct Chains{T<:Real} <: AbstractChains
+struct Chains{T} <: AbstractChains
+    value::AxisArray
```

Comment:
I think it would be better to specify the type, e.g. `AxisArray{Union{Missing, T},3}`.